version/version.go

```diff
@@ -23,7 +23,7 @@ var (
 	Package = "github.com/containerd/containerd"
 
 	// Version holds the complete version number. Filled in at linking time.
-	Version = "1.7.0-rc.3+unknown"
+	Version = "1.7.0+unknown"
```

Comment:
Still wishing for this one to be "without `+unknown`" for the actual tag, and `+unknown` for anything that comes after 😅

Reply:
We can change this in 2.0